File: nj2k/testData/newJ2k/assignmentExpression/nullability-settings.java
// !FORCE_NOT_NULL_TYPES: false
// !SPECIFY_LOCAL_VARIABLE_TYPE_BY_DEFAULT: true
import java.util.HashSet;

class Foo {
  void foo(HashSet o) {
    HashSet o2 = o;
    int foo = 0;
    foo = o2.size();
  }
}

File: j2k/testData/fileOrElement/assignmentExpression/nullability-settings.kt
// !FORCE_NOT_NULL_TYPES: false
// !SPECIFY_LOCAL_VARIABLE_TYPE_BY_DEFAULT: true
import java.util.HashSet

internal class Foo {
    fun foo(o: HashSet<*>?) {
        val o2: HashSet<*>? = o
        var foo: Int = 0
        foo = o2!!.size
    }
}